src/implementations/twig/components/media-container/media-container.html.twig

```diff
@@ -106,7 +116,14 @@
 
 {# Print the result #}
 
-<div class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+<div class="{{ _css_class }}"{{ _extra_attributes|raw }} 
```

Comment:
Maybe i just fail to understand this, but these two separate parameters are actually going to be concatenated in a pretty elaborated and risky way by the js. So why do we need to parameters? As far as i can see they are simply never used individually, we build in a complicated way an aria-label combining the two, and we provide two separate html attributes for the js to combine their value and set a title in an iframe.
I do not understand:

- why do we have two parameters
- why, if we combine these two string in the template to generate an aria-label then we expect the js to retrieve the two values independently and re-combine the same string (hopefully, cause those strings might get escaped).
- so, to me we could/should use one parameter only and use that in the html attr, with all the risks involved by setting a string as the value of an html attribute, ther would not be any need to combine those strings and surely not to replicate the same procedure in js.

Forgive me if i missed something..

Reply:
That's interesting. What you say is true indeed for the media container: we have 2 fields and only use them in the iframe title.
But there are a few other elements to be considered:
- currently it is the same thing on the gallery (2 different fields), the main difference being that the title is also displayed on the screen there
- the title in media container may or may not be used later for something else (this is not limited to videos)

So I would say that it's up to us to decide: keep the same structure for gallery and media container (even if that's a bit useless for now on the media container), or use a single field only in the media container

Reply:
And let's keep the two parameters, but at least we combine this string in twig only and we don't duplicate this in the js